devito/archinfo.py

```diff
@@ -40,7 +40,9 @@ def get_cpu_info():
 
         def get_cpu_flags():
             try:
-                flags = [i for i in lines if i.startswith('flags')][0]
+                # ARM Thunder X2 is using 'Features' instead of 'flags'
+                flags = [i for i in lines if (i.startswith('Features')
+                         or i.startswith('flags'))][0]
```

Comment:
nitpicking: the `or i.startswith('flags)'` should be aligned with `i.startswith('Featuers')`